agent/handlers/task_server_setup_sudo_integ_test.go

```diff
@@ -0,0 +1,427 @@
+//go:build linux && sudo
+// +build linux,sudo
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package handlers
```

Comment:
Can we not add this test to `ecs-agent` module since the Fault Injection handlers that we are integ testing are defined there?

Reply:
+1

Reply:
We can...although the thought process was that this is essentially copying what we're doing in task_server_setup.go

Reply:
I'll need to spend some time refactoring the code and moving it into `ecs-agent` in that case

Reply:
it's been moved